en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,627 @@
+# Get Started with Choreo
+
+Choreo is a cloud-native engineering platform that allows you to develop integration solutions by designing APIs, API Proxies, webhooks, triggers, and scheduled tasks.
+
+This guide walks you through the following:
+
+- Developing, deploying, testing, and publishing a REST API. 
+
+- How an external application consumes the published REST API.
+
+- How to release new versions of the REST API with added functionality.
+
+Let's consider a use case where a developer named Jane designs an application for reading lists. A Choreo developer named John develops a REST API for this application to consume. The two developers will develop three versions of the solution in three iterations. Each new version will introduce new functionality as described below:
```

Comment:
I didn't find any reference to these two characters in the guide after this. If we are going ahead with this approach it would make sense for us to mention this is what Jane has to do, this is what John has to do. I think here Jane is a Backend API developer. John is the API consumer and the frontend developer.